ui-tests/src/test/java/org/openmrs/reference/page/ManageUserPage.java

```diff
@@ -54,7 +55,16 @@ public void assignRolesToUser(String roleToUnassign, String roleToAssign, String
         editPage.clickOn(By.id(roleToAssign));
         editPage.clickOn(SAVE_BUTTON);
     }
-
+    
+    public void findUser(String findUser) {
+    	  setText(FIND_USER, findUser);
+    	  clickOn(ACTION);
+    }
+    
+    public void clickOnAdminAfterSearch() {
```

Comment:
I'd also rename this to `clickOnFirstSearchResult()` or something similar.

Reply:
Resolved thanks